repository-data/application/src/main/resources/hcm-config/namespaces/hee/blogPage.yaml

```diff
@@ -299,7 +299,7 @@ definitions:
           /rightHandBlocks:
             jcr:primaryType: frontend:plugin
             caption: Right hand objects
-            compoundList: hee:contactCardWithDescriptionReference,hee:externalLinksCardReference,hee:fileLinksCardReference,hee:internalLinksCardReference,hee:QuickLinks
+            compoundList: hee:contactCardWithDescriptionReference,hee:ctaCardReference,hee:externalLinksCardReference,hee:fileLinksCardReference,hee:internalLinksCardReference,hee:QuickLinks
```

Comment:
Could you add the rendering support for `hee:ctaCardReference` on `repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/blogpost-main.ftl`:
```
<#case "uk.nhs.hee.web.beans.CtaCardReference">
    <@hee.ctaCard ctaCard=block/>
    <#break>
```

Reply:
Done